system-tests/helm/README.md

```diff
@@ -0,0 +1,115 @@
+# Helm Charts for MVD
+We have demonstrated a containerized deployment of the MVD in [system-tests/README.md](../README.md). To take it one step further
+we will be using [kubernetes](https://kubernetes.io/docs/home/) which is a container orchestration tool for automated deployment, scaling, and management of these containers.
+In addition, we have used [Helm](https://helm.sh/docs/) to manage all the Kubernetes YAML files.
+[./helm-charts](./helm-charts) folder contains the helm charts for the kubernetes components of the applications used in MVD.
+
+
+## Deploying MVD
+For the deployment purpose we need,
+* a kubernetes cluster, and
+* helm installed
+
+You can use [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/)
+to set up a local kubernetes cluster for this testing purpose. Follow the [documentation](https://minikube.sigs.k8s.io/docs/start/)
+to install minikube in your machine. Once everything is installed, you can start the cluster with ```minikube start``` command.
+Also, we will be using Helm, which can be installed following the [instructions](https://helm.sh/docs/intro/install/) provide in their official website.
+
+
+## Mounting Resources
+All the necessary resources needed by these applications are in the directory [./k8s_resources](./k8s_resources).
```

Comment:
nitpick(non-blocking): "these applications", which ones?

Reply:
This line has been omitted in the [latest commit](https://github.com/FraunhoferISST/edc-mvd/blob/helm-with-dashboard-changes/system-tests/helm/README.md).